API_compatible_README.md

```diff
@@ -0,0 +1,24 @@
+# API Compatible
+
+*Instructions are for setting up API for Audacious client on Ubuntu. The steps can be modified for other clients as well.*
+
+**SOURCE_1**: http://redmine.audacious-media-player.org/boards/1/topics/788
+
+1. Install dependencies from **SOURCE_1** then clone the repo and install audacious.
+2. Before installing audacious-plugins, edit the file `audacious-plugins/src/scrobbler2/scrobbler.h` to update the following settings on lines L26-L28.  
+  - `SCROBBLER_API_KEY` to the key given on "listenbrainz.org/user/import" page.
+  - `SCROBBLER_URL` to "http://last.fm/2.0/".  
```

Comment:
- Updated the instructions to prevent changing source code.
- Made changes to the api_key logic in 0fa32e0112b67982dc1fb41e6238a4ddd6721802